lakefile.lean

```diff
@@ -11,6 +11,13 @@ lean_lib Mathlib where
     "-Dpp.unicode.fun=true" -- pretty-prints `fun a ↦ b`
   ]
 
+lean_lib Mathlib.Mathport.Rename where
```

Comment:
@tydeu This seems to work fine out of the box, but are there any guarantees about what happens when a module is included in multiple `lean_lib`s?

Reply:
@Kha I do not think so? It currently searches an `Array` constructed from a `fold` of a `RBMap`, so the order is up in the air. This could be solved in the future with some form of `OrdRBMap` that preserves ordering to retain the ordering of the definitions in the file.

Reply:
For future reference, a strict order of libraries has been established and the last one now wins (lean4#2937).